plugins/kubernetes/test/models/kubernetes/build_job_executor_test.rb

```diff
@@ -128,6 +128,44 @@ def execute!(push: false, registry_info: nil)
           assert_empty job_log
         end
 
+        it 'tests clair exit code 0 - success' do
```

Comment:
being more descriptive makes diving into this later easy:

`it "prints results when clair scan passes"`
`it "shows errors when clair scan fails"`